src/kudu/util/oid_generator.cc

```diff
@@ -31,15 +32,23 @@
 using std::string;
 using strings::Substitute;
 
+DEFINE_bool(cononicalize_uuid, true,
```

Comment:
typo: canonicalize, also let's call it canonicalize_uuids

I know this is a little pedantic but we could probably upstream this patch

Reply:
I am looking for someone better at this to do it.